packages/mobile/src/components/skeleton/Skeleton.tsx

```diff
@@ -51,12 +52,13 @@ const Skeleton = ({ width, height, style }: SkeletonProps) => {
         })
       ).start()
     }
-  }, [shimmerPos, shimmerWidth])
+  }, [shimmerPos, shimmerWidth, noShimmer])
 
   return (
     <View style={[styles.view, { height, width }, style]}>
       <Animated.View
         onLayout={(e) => {
```

Comment:
should we just not provide an onLayout at all?

Reply:
i think we should look into this. it seems like this is needed for cases where width is not specified. we can update our usages to ensure we always explicitly set it. seems nice to allow flex to help us sometimes tho.